packages/protocol/test/tokenomics/Tokenomics.test.ts

```diff
@@ -0,0 +1,298 @@
+import { expect } from "chai";
+import { BigNumber, ethers } from "ethers";
+import { ethers as hardhatEthers } from "hardhat";
+import { TaikoL1, TaikoL2, TkoToken } from "../../typechain";
+import { TestTkoToken } from "../../typechain/TestTkoToken";
+import Proposer from "../utils/proposer";
+import createAndSeedWallets from "../utils/seed";
+import sleep from "../utils/sleep";
+import { defaultFeeBase, deployTaikoL1 } from "../utils/taikoL1";
+import { deployTaikoL2 } from "../utils/taikoL2";
+import deployTkoToken from "../utils/tkoToken";
+import sendTransaction from "../utils/transaction";
+
+describe("tokenomics", function () {
+    let taikoL1: TaikoL1;
+    let taikoL2: TaikoL2;
+    let l1Provider: ethers.providers.JsonRpcProvider;
+    let l2Provider: ethers.providers.JsonRpcProvider;
+    let l1Signer: any;
+    let l2Signer: any;
+    let proposerSigner: any;
+    let proverSigner: any;
+    let genesisHeight: number;
+    let genesisHash: string;
+    let tkoTokenL1: TestTkoToken;
+
+    beforeEach(async () => {
```

Comment:
prefer to use fixtures as it's much faster (no execution)

Reply:
I think we've mvoed away from fixtures to having a `beforeEach` setup, and we had Roger remove all the fixtures.